### ISSUE TITLE
Upload failure

### ISSUE THREAD

**Issue by UpcraftLP** [matthewprenger/CurseGradle#22]
CG 1.1.1 fails to upload jars to curseforge:
```
Execution failed for task ':curseforge308081'.
> Could not get unknown property 'ContentType' for task ':curseforge308081' of type com.matthewprenger.cursegradle.CurseUploadTask.
```

here's my buildscript: https://github.com/UpcraftLP/Olaf/blob/fc9aa7e818c70ef1b74353a3ba0cf6181921c3cc/build.gradle

**Comment by matthewprenger** [comment]
This should be fixed in 1.1.2.

I blame @dshadowwolf for missing an import in 9371e66ae2c14c2abc6667659b0d17d78dfaa5d6. Unfortunately debug mode (how I test) didn't get far enough in the code path to hit this.

**Comment by UpcraftLP** [comment]
alright, thank you ^^
